tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py{27,33,34}-{cdh,hdp,nonhdfs,gcloud,postgres}, pypy-scheduler, docs, pep8
+envlist = py{27,33,34}-{cdh,hdp,nonhdfs,gcloud,postgres}{-requests,-unix,}, pypy-scheduler, docs, pep8
```

Comment:
I don't see any corresponding `attr('requests')`, nor did you add it to the `.travis.yml` right? Also i don't see why you need multiplicative expansion here. Why not just `py27-unix`? Also something more explicit over just `unix`. Like `py27-unixsockets` or something.